ckanext/saml2auth/plugin.py

```diff
@@ -108,6 +108,11 @@ def logout(self):
                 # CKAN <= 2.9.x also sets auth_tkt cookie
                 response.set_cookie('auth_tkt', domain=domain, expires=0)
 
+        if g.userobj is not None:
+            log.info("Logout successful %s<%s>" % (g.userobj.name, g.userobj.email))
+        else:
+            log.info("No user was logged in!")
```

Comment:
From a security standpoint, it seems a bit strange that the logout route would be hit multiple times with no user logged in.  It would just be for validation that no work was done.  However, if it seems extraneous, I have no issues removing it.

Reply:
Let's go with it, no issues. :)